clipper_admin/clipper_admin/sample_config.yaml

```diff
@@ -0,0 +1,42 @@
+models:
```

Comment:
This file shouldn't be in clipper_admin. You should create an example in the [`examples/`](https://github.com/ucbrise/clipper/tree/develop/examples) directory that includes both the sample config file and an example Python file using it.